test/evergreen.yml

```diff
@@ -1783,6 +1804,34 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger sanitizer"
+      - func: "format test script"
+        vars:
+          test_env_vars: ${test_env_vars|} ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
+          # run for 24 hours ( 24 * 60 = 1440 minutes), stop at first error, use default config
+          format_test_script_args: -F -t 60
+  
+  - name: format-stress-sanitizer-smoke-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger sanitizer"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/test/format"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+            # to emulate the original Jenkins job's test coverage, we are running the smoke test 16 times
+            # run smoke tests, stop at first error, use default config (-F -S)
+            for i in {1..2}; do
```

Comment:
Ignore this argument now. I am running a seperate patch build to verify the actual arguments.